translations/sk.json

```diff
@@ -158,7 +158,7 @@
   "ParkingCards.type.subscriber": "Abonentská karta",
   "ParkingCards.type.tzp": "ŤZP karta",
   "ParkingCards.type.visitor": "Návštevnícka karta",
-  "ParkingCards.validUntil": "Platnosť do",
+  "ParkingCards.validUntil": "Platnosť do (vrátane)",
```

Comment:
I'm against this wording. It's redundant and we should not introduce it anywhere. As we discussed, the date is always considered "vratane" in SK laws.